### PR TITLE
Add documentation for core profiler and memory profiler in VSCode ext…

### PR DESCRIPTION
**Pull request by adhavan-m**
…ension

### Change Description ###

Describe your changes here.

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

File: docs/application/vscode-ext/tools/profiler-user-manual.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+     ![Launch Emulator](media/start_emulator.png)
+
+
+   - Open the **Command Palette** and select **Tizen.NET: Run Tizen core profiler**.
+
+     ![Run Memory Profiler](media/run_profiler.png)
+
+
+   - If everything is OK then the application starts as if you run it normally.
```
Comment (omar-srbd) [inline docs/application/vscode-ext/tools/profiler-user-manual.md:14]:
```suggestion
   - If everything is ok then the application starts as if you are running it normally.
```

File: docs/application/vscode-ext/tools/profiler-user-manual.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+     ![Tizen application running](media/profiling_app_started.png)
+
+
+   - Profiling progress window will be displayed with stop button in a new tab which shows live performance profiling of the application.
```
Comment (omar-srbd) [inline docs/application/vscode-ext/tools/profiler-user-manual.md:19]:
```suggestion
   - Profiling progress window will be displayed with a "Stop" button in a new tab showing the application's live performance profiling.
```

File: docs/application/vscode-ext/tools/profiler-user-manual.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+     ![Close Application](media/profiling_progress.png)
+
+   - Profiling can be stopped by clicking stop button.
```
Comment (omar-srbd) [inline docs/application/vscode-ext/tools/profiler-user-manual.md:23]:
```suggestion
   - Profiling can be stopped by clicking the "Stop" button.
```

**Review by omar-srbd — COMMENTED** [review]
@adhavan-m hello, I have reviewed the PR and left some correction suggestions for you.

Please check and incorporate them.

Thank you.

**Comment by adhavan-m** [comment]
Hello @omar-srbd ,
I have applied your suggestions. Please have a look

File: docs/application/vscode-ext/tools/memory-profiler-user-manual.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+## Analyze the results
+   - The GUI application provides several views of the memory profiling data. The views include the following:
+
+       - Summary page with information on which process was profiled, its total runtime, some memory related statistics, and so on.
+       - Bottom-up table tree view of the code locations that allocated memory with their aggregated cost and stack traces.
+       - Caller/Callee table.
+       - Top-down table tree view of the code locations.
+       - Managed heap table tree view.
+       - Flame graph visualization (explanation: (http://www.brendangregg.com/FlameGraphs/memoryflamegraphs.html){:target="_blank"})
```
Comment (omar-srbd) [inline docs/application/vscode-ext/tools/memory-profiler-user-manual.md:31]:
```suggestion
       - Flame graph visualization (explanation: [http://www.brendangregg.com/FlameGraphs/memoryflamegraphs.html](http://www.brendangregg.com/FlameGraphs/memoryflamegraphs.html){:target="_blank"}).
```

File: docs/application/vscode-ext/tools/memory-profiler-user-manual.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+       - Consumed memory size over time graph.
+       - Number of instances over time graph.
+       - Number of memory allocations over time graph.
+       - Size of memory allocated over time graph.
+       - Allocation histogram displaying the number of allocations (the total number and the several topmost code locations) belonging to one of the groups divided by allocation size (0 - 8 bytes, 9 - 16 bytes, ... , 512 bytes - 1 KB, more than 1 KB)
```
Comment (omar-srbd) [inline docs/application/vscode-ext/tools/memory-profiler-user-manual.md:36]:
```suggestion
       - Allocation histogram displaying the number of allocations (the total number and the several topmost code locations) belonging to one of the groups divided by allocation size (0 - 8 bytes, 9 - 16 bytes, ... , 512 bytes - 1 KB, more than 1 KB).
```

**Review by omar-srbd — COMMENTED** [review]
Hello @adhavan-m,

Please check the suggestions.

Thank you.

File: docs/application/vscode-ext/index.md [hunk 6]
```diff
@@ -22,6 +22,10 @@ Visual Studio Code Extension for Tizen enables you to develop Tizen .NET and Tiz
 
     Download from <https://nodejs.org>.
 
+  - Python 2.7 (only for Mac OS)
```
Comment (omar-srbd) [inline docs/application/vscode-ext/index.md:25]:
```suggestion
  - Python 2.7 (only for macOS)
```

**Review by omar-srbd — COMMENTED** [review]
@adhavan-m, Please check the suggestion.

Thank you.

**Comment by omar-srbd** [comment]
Hello @adhavan-m, this PR has been inactive for a while.
Please check the suggestions and incorporate them.

Thank you.

**Comment by adhavan-m** [comment]
> Hello @adhavan-m, this PR has been inactive for a while. Please check the suggestions and incorporate them.
> 
> Thank you.

Hi @omar-srbd I have incorporated all your comments, please check the latest changes and notify if any more change is required

**Review by omar-srbd — APPROVED** [review]
@adhavan-m, thanks for applying all the changes as per the suggestion.

@asm-foysal, I am approving this PR for merging.

**Pull request merged**